src/main/java/net/twasi/obsremotejava/message/event/highvolume/InputVolumeMetersEvent.java

```diff
@@ -18,13 +19,12 @@ protected InputVolumeMetersEvent() {
     @Getter
     @ToString
     public static class Data {
-        private List<Input> inputs;
+        private List<InputLevels> inputs;
     }
 
     @Getter
     @ToString
```

Comment:
`@ToString(callSuper = true)`

Reply:
Yep, nothing else to add here. I'll go ahead and approve, so it's just your approval left.